Python/Product/TestAdapter/testlauncher.py

```diff
@@ -33,7 +33,7 @@ def load_debugger(secret, port, debugger_search_path, mixed_mode):
     # Load the debugger package
     try:
         if debugger_search_path:
-            sys.path.append(debugger_search_path)
+            sys.path.insert(1, debugger_search_path)
```

Comment:
Is there any particular reason why this is inserted as the 2nd item in the search path, rather than the 1st (index 0)?

Reply:
It's convention that sys.path[0] is the directory from which the script was invoked. 

Stackoverflow:
![image](https://user-images.githubusercontent.com/3315992/71920097-e9e22e80-313a-11ea-825b-b606d0bbd983.png)


https://stackoverflow.com/questions/10095037/why-use-sys-path-appendpath-instead-of-sys-path-insert1-path/10095099#10095099

Reply:
I didn't spot this before, but this code should be removing the entry after importing all the top-level packages that it needs - there's no reason for it to be around there afterwards (if user re-imports explicitly, it's already in `sys.modules`).

The problem with leaving `path[0]` alone is that it may still interfere, e.g. if they have a local copy of ptvsd around.

Reply:
Will be updated in next commit